ansible/roles/ocp4-workload-dil-streaming/defaults/main.yaml

```diff
@@ -21,14 +21,15 @@ mssqlserver_image: microsoft/mssql-server-linux
 mssqlserver_version: 2017-CU9-GDR2
 
 # provision_webapp
-webapp_operator_release_tag: '0.0.45'
+webapp_operator_release_tag: '0.0.63'
 webapp_client_id: tutorial-web-app
+ocp4_workload_dil_serverless_webapp_group_name: dedicated-admins
 webapp_operator_template_path: /home/tutorial-web-app-operator/deploy/template/tutorial-web-app.yml
 webapp_operator_resources: 'https://github.com/integr8ly/tutorial-web-app-operator/archive/v{{webapp_operator_release_tag}}.zip'
 webapp_operator_resource_items:
   - rbac.yaml
   - sa.yaml
   - crd.yaml
-  - operator.yaml
+  # - operator.yaml
```

Comment:
Please remove this vs leaving it commented out.

Reply:
done